ci-operator/step-registry/cucushift/ota/postupgrade/cucushift-ota-postupgrade-commands.sh

```diff
@@ -179,25 +179,8 @@ function post-OCP-47197(){
 
 function post-OCP-53921(){
     echo "Test Start: ${FUNCNAME[0]}"
-    local arch version
-    arch=$(oc get clusterversion version -ojson|jq -r '.status.conditions[]|select(.type == "ReleaseAccepted")|.message')
```

Comment:
To identify related case for the failure easily, could u update the msg to include case id accordingly in https://github.com/openshift/release/blob/master/ci-operator/step-registry/cucushift/upgrade/arch-migration/cucushift-upgrade-arch-migration-commands.sh#L324

```
"OCP-53921: ClusterVersion architecture check FAILED, exiting"
```

Reply:
It looks like you updated above msg while keeping the arch check in the case in https://github.com/openshift/release/compare/774c8d6ec209edb355c058f66f6ec93aebd5887a..f0b2345615934c49f9b01e32c9b881c655890138? Based on the discussion, the arch check part could be removed since it's covered in the migrate chain?